plugins/auth-backend/src/providers/types.ts

```diff
@@ -122,7 +122,7 @@ export type ExperimentalIdentityResolver = (
    * An object containing information specific to the auth provider.
    */
   payload: object,
-  catalogApi: CatalogApi,
+  catalogClient: CatalogClient,
```

Comment:
hm why was this change made? you want to pass in the interface instead of the concrete type